ooniprobe/View/BrowserViewController.m

```diff
@@ -1,10 +1,6 @@
-//
-//  BrowserViewController.m
-//  ooniprobe
-//
-//  Created by Lorenzo Primiterra on 23/06/17.
-//  Copyright © 2017 Simone Basso. All rights reserved.
-//
+// Part of MeasurementKit <https://measurement-kit.github.io/>.
+// MeasurementKit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
```

Comment:
Note to self: after this release cycle, these comments should actually be updated to reference OONI. Since I already have a script for dealing with that in MK, I can apply it also to ooniprobe-ios and -android.